### PR TITLE
Added Unacademy Userstyle

### PR DESCRIPTION
**Pull request by kushdhingra**
## 🎉 Theme for Website 🎉

Soothing Pastel Theme for an educational tech platform called unacademy
image
image
image

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [ ] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [ ] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

**Comment by uncenter** [comment]
Are you sure this is ready to review? Looks like you've missed some tickboxes there.

File: styles/unacademy/catppuccin.user.css
Comment (uncenter) [inline styles/unacademy/catppuccin.user.css:1]: Looks like this isn't following our userstyles template either, meaning it only works for one flavor. Please see https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md#creating-userstyles.

Comment (kushdhingra) [reply styles/unacademy/catppuccin.user.css:1]: I dont know less and preview.webp is not used, so ya you can submit!

**Comment by uncenter** [comment]
Seeing as you aren't willing to conform to the userstyles standard as established in the documentation we can't accept this.

**Comment by kushdhingra** [comment]
¯\_(ツ)_/¯

**Pull request closed without merging**